src/modules/dashboard/components/ActionsPage/MultisigWidget/MultisigWidget.css

```diff
@@ -1,7 +1,38 @@
 @value defaultDotBackground: rgb(255, 176, 0);
 
 .wrapper {
-  margin-top: 20px;
+  margin-top: 30px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+}
+
+.title > div {
+  margin: 0 6px;
+}
+
+.avatars {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 16px;
+}
+
+.avatarWrapper {
+  margin: 0 13px 10px 0;
+}
+
+.avatarWrapper:nth-child(7n) {
+  margin-right: 0px;
+}
+
+.avatarWrapper figure {
+  height: 37px;
+  width: 37px;
```

Comment:
I think we need to finally address this avatar refactor to keep only two sizes - 37x37 and 28x28 as we have more and more different avatar sizes across the app (and overriding defaults from avatar component)

Reply:
Agree. However, for now, I'll keep this sized manual.